qttypes/Cargo.toml

```diff
@@ -40,6 +40,7 @@ chrono = { version = "0.4", optional = true }
 
 [build-dependencies]
 cpp_build = "0.5.6"
+semver = "1"
```

Comment:
i'm a bit disappointed by adding this dependencies just for detecting Qt6.  it's a bit overkill and will not help with compile time.

Reply:
I checked it, and it's 80K of Rust files, mostly doc comments, serde support and tests (OMG so many tests). After stripping it down, it's about 30K of code, no generics. Still a bit of an overkill, especially comparing to C++ `#define QT_VERSION_CHECK()`, but oh well… that's what rust it all about :)